migrations/sql/V2020.08.04.15.09__add_template_permit_conditions.sql

```diff
@@ -0,0 +1,104 @@
+DROP SEQUENCE standard_permit_conditions_standard_permit_condition_id_seq cascade;
+
```

Comment:
Removing the Serial from PK, this appears to work fine, but please advise on a better approach if there is one.